draft-ietf-cose-x509.xml

```diff
@@ -208,26 +208,17 @@ obtained from any of these methods still need to validate it.
             <li>application/pkcs7-mime; smime-type="certs-only" <xref target="RFC8551"/></li>
-            <li>application/cbor <xref target="RFC8949"/></li>
+            <li>application/cose-x509-chain <xref target="RFC8949"/></li>
             <!--              <t>application/pem-certificate-chain <xref target="I-D.ietf-acme-acme"/></t> -->
           </ul>
           <t>
-            When the application/cbor media type is used, the data is a COSE_X509 structure containing a chain.
+            When the application/cose-x509-chain media type is used, the data is a COSE_X509 structure containing a chain.
           </t>
           <t>
-		Unless it is known to both sender and recipient that proof-of-possession of the subject's private key was required for certificate issuance, the end-entity certificate MUST be integrity protected by COSE. This can e.g. be done by sending the x5u in the unprotected or protected header combined with a x5t in the protected header, or including the end-entity certificate in the external_aad. 
-          </t>
-	  <t>
-	     If the end-entity certificate is integrity protected by COSE, the URI does not need to provide any protection.
-          </t>
- 	<t>
-	If the end-entity certificate is not integrity protected by COSE, this header parameter implies a trust relationship between
-	the party generating the x5u parameter and the party hosting the referred-to resource, this header parameter MUST then be in the
-	protected attribute bucket and the URI MUST provide integrity protection and server authentication. For example, an HTTP or CoAP
-	GET request to retrieve a certificate MUST use TLS <xref target="RFC8446"/> or DTLS <xref target="I-D.ietf-tls-dtls13"/>.
+	   The end-entity certificate MUST be integrity protected by COSE. This can e.g. be done by sending the x5u in the unprotected or protected header combined with a x5t in the protected header, or including the end-entity certificate in the external_aad. As the end-entity certificate is integrity protected by COSE, the URI does not need to provide any protection.
           </t>
  	<t>
-            If a retrieved certificate does not chain to an existing trust anchor, the certificate MUST NOT be trusted unless the URI provided integrity protection and server authentication and the server is configured as trusted to provide new trust anchors or if an out-of-band confirmation can be received for trusting the retrieved certificate.
+            If a retrieved certificate does not chain to an existing trust anchor, the certificate MUST NOT be trusted unless the URI provided integrity protection and server authentication and the server is configured as trusted to provide new trust anchors or if an out-of-band confirmation can be received for trusting the retrieved certificate. For example, an HTTP or CoAP GET request to retrieve a certificate MUST use TLS <xref target="RFC8446"/> or DTLS <xref target="I-D.ietf-tls-dtls13"/>.
```

Comment:
This clarification seems anti-helpful.  The existing sentence tries to emphasize that the server is specifically trusted to provide new trust anchors, but the new sentence is just talking about "retrieve a certificate" with no explicit mention of authenticating the server or that the authenticated server is trusted to provide trust anchors.

Reply:
None of these sentences are new. The only change I made was to change to order of "If a retrieced" and "For example". The reason for this is that after the addition of The end-entity certificate MUST be integrity protected by COSE, the "MUST use TLS or DTLS" is no longer general but specific for the case "If a retrieved certificate". 

It seems like you suggest deleting the sentence about TLS and DTLS. Do you still want to do that even if the text is not new?